import 'package:part_eight_exceptions/part_eight_exceptions.dart';
import 'package:test/test.dart';

void main() {
  test('calculate', () {
    expect(calculate(), 42);
  });
}
